code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -359,28 +359,23 @@
 			popDown()
 		return
 
-	var/list/targets = calculate_targets()
-
-	if(!tryToShootAt(targets))
-		if(!always_up)
-			popDown() // no valid targets, close the cover
-
-/obj/machinery/porta_turret/proc/calculate_targets()

```

Comment:
I decided to not keep this a proc to cut down on proc call overhead, since this isn't used anywhere else.